eno-core/src/test/java/fr/insee/eno/core/processing/out/steps/lunatic/LunaticAddCleaningVariablesTest.java

```diff
@@ -0,0 +1,380 @@
+package fr.insee.eno.core.processing.out.steps.lunatic;
+
+import fr.insee.eno.core.model.lunatic.CleaningConcernedVariable;
+import fr.insee.eno.core.model.lunatic.CleaningVariable;
+import fr.insee.lunatic.model.flat.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LunaticAddCleaningVariablesTest {
+
+    LunaticAddCleaningVariables processing;
+    Input i;
+    InputNumber ln;
+    CheckboxOne co;
+    Dropdown ldd;
+    Table t;
+    CheckboxGroup lcg;
+    Loop l;
+
+    Questionnaire lunaticQuestionnaire;
+
+    @BeforeEach
+    void init() {
+        i = buildInput("jfazww20", "TEXTECOURT");
+        co = buildCheckboxOne("k6gik8v5", "CHECKBOX");
+        t = buildTable("jfkxybfe", List.of("QCM_OM1", "QCM_OM2", "QCM_OM3"));
+        ldd = buildDropdown("jfjfae9f", "DROPDOWN");
+        ln = buildNumber("jfjh1ndk", "INTEGER");
+        lcg = buildCheckboxGroup("jfkxybff", List.of("QCM_OM1", "QCM_OM2", "QCM_OM3"));
+        l = buildLoop("loopid", List.of(ldd, ln));
+        processing = new LunaticAddCleaningVariables();
+    }
```

Comment:
(style) The test class would be easier to read with full names for these local variables (inputComponent, checkboxOne, table, dropdown etc.)